### DISCUSSION TITLE
[Prettier Plugin] Why no eslint plugin?

### DISCUSSION THREAD

**Discussion by joshuajaco** [tailwindlabs/tailwindcss#11998] [Help]
To me, sorting of elements is very obviously not a formatter concern, so why is it a formatter plugin?
I think it much rather fits the concern of a linter, which is why I will be using https://github.com/francoismassart/eslint-plugin-tailwindcss instead.
However, I am still curious as to why it was implemented as a prettier plugin in the first place.

**Comment by JasoGreen** [comment]
Hy there i can see your post and i must say

Customization: Developers often customize ESLint configurations and rules to fit their project's specific requirements. Therefore, the need for a plugin may vary from project to project.

Plugin Availability: There are numerous ESLint plugins available for various purposes, such as enforcing coding standards, catching common programming mistakes, and more. The availability of a specific plugin depends on the use case and the needs of the project. You can search for ESLint plugins on the npm registry or GitHub to find plugins that suit your needs. https://www.militarystarcard.net/

Thanks and regards
JasoGreen

**Comment by mybearworld** [comment]
1. An ESLint plugin wouldn't work with simple HTML 
2. Why would sorting the classes be a linter's job? It's literally only to make the code more readable - it doesn't change the code itself at all. That's what formatters do.
